### ISSUE TITLE
Improve UI/User feedback when adding Analytics objects

### ISSUE THREAD

**Issue by bjackson27** [google/site-kit-wp#183]
When configuring Analytics:

![c7bc8e7ee71a7a7e6d4d0ae1d465f5a9](https://user-images.githubusercontent.com/28811945/61000325-ccb2db80-a322-11e9-9ae9-f8008007768c.png)

All three dropdowns: Account, Property and Profile enable selecting a 'Setup a New'... option to create a new entity.

Expected Behavior
* When adding an entry, the UI should provide some feedback, since creating a new entry can take tens of seconds
* The UI should offer a way for the user to check if the new item is available, or automatically populate the new entry when available.

Actual Behavior
* When selecting the new account option, the UI changes and presents a 'refetch' option:

![c7dfdb250523b0f186f060e8b8bef3dc](https://user-images.githubusercontent.com/28811945/61000341-d2a8bc80-a322-11e9-985c-e5f2efe1ebdc.png)

Remedy: We should a descriptive text here helping the user understand what is happening. Re-fetch is confusing.

However, once I select my new account and create a new property:

There is no indication anything is happening and I have to clear my session storage and refresh to see the newly added property. There is no refetch option.

Remedy: Include the same descriptive test and refetch option

Also: the current re-fetch feature doesn't appear to work correctly.

Two issues here:

1) Setting expectations about what will happen
This should be addressed with the copy changes in https://tenup.teamwork.com/#/tasks/18594568
Once the new copy is in place, we won't offer the user any drop-downs to select from; instead, the UI will say:
"Looks like you don't have an Analytics account yet. Once you create it, click 'Re-fetch my account' and Site Kit will locate it."

2) The remaining issue is the time-lag on re-fetching and the necessity of re-fetching at all.
@adams had suggested "Automatically populate the new entry when available" --> can we investigate if that is feasible? How soon after the user creates a new account in GA for that URL can we know that it exists?

If we can do the re-fetching on behalf of the user and automatically display the new account without them having to click "re-fetch", that would be a much better experience.

Ideal flow:
1. User starts GA setup
2. We detect no tag and no account --> show "Looks like you have no account ..." copy + Create account button that goes to GA
3. We detect as soon as the new account is created and display the correct account /property/profile fields
4. User clicks "configure" --> done.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by marrrmarrr** [comment]
This is no longer relevant with the work on GA account provisioning.
